docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -364,3 +368,5 @@ Then, add to the `build` section the `os-maven-plugin` extension and the `protob
 That's why we use the `os-maven-plugin` to target the executable compatible with the operating system.
 
 NOTE: This configuration instructs the `protobuf-maven-plugin` to generate the default gRPC classes and classes using Mutiny to fit with the Quarkus development experience.
+
+IMPORTANT: When using `protobuf-maven-plugin`, every time you update the `proto` files, you need to re-generate the classes (using `mvn compile`).
```

Comment:
```suggestion
IMPORTANT: When using `protobuf-maven-plugin` instead of the `quarkus-maven-plugin`, every time you update the `proto` files, you need to re-generate the classes (using `mvn compile`).
```